presto-orc/src/main/java/com/facebook/presto/orc/metadata/Stream.java

```diff
@@ -21,21 +21,38 @@
 
 public class Stream
 {
+    public enum StreamArea {
+        INDEX,
+        DATA,
+    }
+
     public enum StreamKind
     {
-        PRESENT,
-        DATA,
-        LENGTH,
-        DICTIONARY_DATA,
-        DICTIONARY_COUNT,
-        SECONDARY,
-        ROW_INDEX,
-        BLOOM_FILTER,
-        BLOOM_FILTER_UTF8,
-        IN_DICTIONARY,
-        ROW_GROUP_DICTIONARY,
-        ROW_GROUP_DICTIONARY_LENGTH,
-        IN_MAP,
+        PRESENT(StreamArea.DATA),
```

Comment:
nit: Can we do static import for `DATA` and `INDEX`?

Reply:
Stream enum already has DATA value, so I left both INDEX and DATA without static imports.